src/Microsoft.Framework.PackageManager/project.json

```diff
@@ -43,9 +43,9 @@
 
     "scripts": {
         "postbuild": [
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-clr-win-x86/bin/lib/Microsoft.Framework.PackageManager",
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/kre-mono/bin/lib/Microsoft.Framework.PackageManager",
-            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/kre-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-clr-win-x86/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnet50/*.* %project:Directory%/../../artifacts/build/dnx-mono/bin/lib/Microsoft.Framework.PackageManager",
+            "%project:Directory%/../../build/batchcopy %project:BuildOutputDir%/Debug/aspnetcore50/*.* %project:Directory%/../../artifacts/build/dnx-coreclr-win-x86/bin/lib/Microsoft.Framework.PackageManager"
         ]
```

Comment:
Just chatted with @ChengTian.  The folder renames will be in another PR, as Wei wants to keep things working with this PR.

:+1: